routing/missioncontrol.go

```diff
@@ -273,15 +273,20 @@ func (m *MissionControl) GetProbability(fromNode, toNode route.Vertex,
 
 // setLastPairResult stores a result for a node pair.
 func (m *MissionControl) setLastPairResult(fromNode,
-	toNode route.Vertex, result TimedPairResult) {
+	toNode route.Vertex, timestamp time.Time,
+	result *pairResult) {
 
 	nodePairs, ok := m.lastPairResult[fromNode]
 	if !ok {
 		nodePairs = make(NodeResults)
 		m.lastPairResult[fromNode] = nodePairs
 	}
 
-	nodePairs[toNode] = result
+	nodePairs[toNode] = TimedPairResult{
```

Comment:
why this change? Looks like the creation of `TimedPairResult` is just moved.

Reply:
squashed with next commit